ocw_data_parser/ocw_data_parser.py

```diff
@@ -310,6 +310,7 @@ def generate_master_json(self):
             "sort_as": self.jsons[0].get("sort_as"),
             "department_number": master_course.split('.')[0] if master_course else "",
             "master_course_number": master_course.split('.')[1] if master_course else "",
+            "master_subject_uids": self.jsons[0].get("master_subject"),
```

Comment:
Can we take this opportunity to rename this field here to something clearer and less problematic? 

```suggestion
            "other_versions_uids": self.jsons[0].get("master_subject"),
```

or `other_versions_collection_uids` if that isn't too long.

Reply:
I thought about that too, but since we can't really do any transformation of this data due to what we discussed earlier with not having the context of the Plone `mastersubject` tree, I think `master_subject_uids` unfortunately makes the most sense here because that's what these uid's specifically refer to, the Plone `id` property on the objects in the `mastersubjects`.  It would be confusing to name it `other_versions_` I think because it's not the uid of another course, rather a completely unrelated object within Plone.

Reply:
@pdpinch Maybe something like `other_version_parent_subjects` would be better?

Reply:
Sorry for the delay. That field name is fine.